spec/server.test.js

```diff
@@ -0,0 +1,20 @@
+const request = require('supertest');
+const app = require('../server/app');
+
+describe('Test the root path', () => {
+  test('It should response the GET method', () =>
+    request(app).get('/').then(response =>
+      expect(response.statusCode).toBe(200)));
+
+  test('It should get a response Hello World back', () =>
```

Comment:
Tests on line 9 and 13 will be obsolete in next feature, most likely. Consider either not including OR including data + paths that match what you actually want to build.